### DISCUSSION TITLE
Breadcrumb

### DISCUSSION THREAD

**Discussion by Lephilosophe** [wet-boew/wet-boew#9373] [Q&A]
I would like to implement a breadcrumb similar to the one done in canada.ca. I would like to get explanations on how I could achieve it. I plan to use only html css and javascript/jquerie to do this. I did a search directly on the canada.ca site to understand how they made theirs but I could only find this https://wet-boew.github.io/GCWeb/sites/breadcrumbs /breadcrumbs-en.html. do I have to map the site in json manually so that I can use it to compare the paths for each page before displaying the breadcrumb ?

**Comment by ghost** [comment]
You have to find a way on the server side to generate them. On GitHub the project uses Jekyll to generate the html.

https://github.com/wet-boew/wet-boew/blob/master/site/includes/breadcrumbs.hbs
https://github.com/wet-boew/GCWeb/blob/master/sites/breadcrumbs/inc-breadcrumbs.html

**Reply by Lephilosophe** [reply]
Thanks for the answer, I will explore this solution.
